src/main/java/org/opensrp/web/rest/ProductCatalogueResource.java

```diff
@@ -63,8 +64,16 @@
 	@PostMapping(headers = { "Accept=multipart/form-data" })
 	public ResponseEntity<String> create(@RequestPart(required = false) MultipartFile file,
 			@RequestPart ProductCatalogue productCatalogue) {
+
+		final String DOWNLOAD_PHOTO_END_POINT = "/multimedia/media/";
```

Comment:
can this be a static variable in this class

Reply:
Done